lib/spoom/snapshot.rb

```diff
@@ -0,0 +1,96 @@
+# typed: strict
+# frozen_string_literal: true
+
+module Spoom
+  class Snapshot < Hash
+    extend T::Sig
+    extend T::Generic
+
+    K = type_member(fixed: String)
+    V = T.type_alias { type_member(fixed: T.untyped) }
+    Elem = type_member(fixed: String)
+
+    sig { params(path: String).returns(Snapshot) }
+    def self.snapshot(path: '.')
+      snapshot = Snapshot.new
+      metrics = Spoom::Sorbet.srb_metrics(path: path, capture_err: false)
+      return snapshot unless metrics
+
+      snapshot["user_time"] = metrics["run.utilization.user_time.us"]
```

Comment:
I hand pick the data so the resulting JSON (later) will be smaller (we don't care about a lot of the data generated by `sorbet --metrics`